docs/currency.md

```diff
@@ -0,0 +1,129 @@
+
+For example, ETH was transferred from Etherium to Acala and to this parachain as AETH. Same time METH got via Moonbeam. Moonbeam was hacked, and minted a huge amount of unbacked METH hacker wants. In this the case DEX(market) would react and skew the price of AETH to be higher than METH. 
+More mild form of non equivalence would be becasue of network congestion. Imagine Acala transfer prices are several time more of Moonbeam. In this case  AETH will be less liquid than METH, so it price may be lower. 
+
+### Remote currency identifiers
+
+Any remote currency has a bidirectional map from and to local.
+Bidirectional mapping allows sending the number of tokens from and to local consensus.
+
+Remote currencies usually have their respective bridge identifiers attached in the format "kind/network(chain)".
+
+## Native currencies and gas fees
+
+Users pay fees in the runtimes native currency for the execution. It allows the runtime to earn and prevents spam and DDoS.
+
+A runtime configuration allows mapping amount of execution which can be bought for amount of native currency.
+
+This parachain uses pre-calculated weights to estimate hardware costs of execution. An amount of weight a native currency can buy is a non-linear dynamic function from currency amount to weight. As the usage of this chain goes to maximum capacity, a unit of weight costs more and more. 
+
+Cross-chain interactions over bridges may involve metered networks which allow setting limits on execution price and calculating the possible number of execution resources in the runtime. For example, Ethereum.
+
+There are non-paying `native` currencies used inside of protocols built on top of the runtime. They are used for governance and revenue distribution but do not pay for execution.
+
+### Bridged gas fee (`bring your own gas`)
+
+Users who want to have assets on our chain can either be minted with some native currency or be allowed to pay for cross-chain transactions in some non-native currency.
+
+Currently, non-native currencies are mapped to native with some ratio which in turn is mapped to a weight.
+
+Transactions issued by users are likely to swap some remote currencies into native on transfer, to enact the execution of local transactions. 
+
+## Tokenomics
+
+So any currency is eventually local, and minting is governed by a protocol.
+
+The most straightforward protocol is initial minting by `root` or `governance voting`.
+
+Other more complicated protocols are `bridges`, `staking`, or `LP` are more complicated.
+
+Protocols form various economic facets of tokens.
+
+### Inflation
+
+In the context of currency, we define inflation as when the amount of an asset can obtain fewer other assets than the same amount of the inflated asset could before.
+
+Inflation may happen when: 
+- Users exchange an inflating currency into other currencies a lot.
+- An excessive amount of tokens are minted to inflate a currency. 
+- Previously locked currency is released to the market.
```

Comment:
```suggestion
- A previously locked currency is released to the market.
```

Reply:
This is referring to the inflated asset having a locked amount released, correct?